tutorials/platform/ios/ios_plugin.rst

```diff
@@ -133,5 +133,5 @@ To build an iOS plugin:
                 -   Each line should follow pattern: ``KeyName:KeyType=KeyValue``
                 -   Supported values for ``KeyType`` are ``string``, ``integer``, ``boolean``, ``raw``, ``string_input``
                 -   If no type is used (e.g.: ``KeyName="KeyValue"``) ``string`` type will be used.
-                -   If ``raw`` type is used value for coresponding key will be stored in ``Info.plist`` as is.
+                -   If ``raw`` type is used the value for the corresponding key will be stored in ``Info.plist`` as is.
```

Comment:
I feel I've opened a can of worms by adding a few "the"s instead of just correcting the spelling mistakes. All your suggestions are valid, but it does highlight that all the documentation needs to be reviewed and edited.